events/python/receive_pika.py

```diff
@@ -0,0 +1,20 @@
+from setup_pika import (
+    DASHBOARD_QUEUE_NAME,
+    get_channel
+)
+
+
+def message_handler(ch, method, properties, body):
+    print(f'[x] Received: {body}')
+
+
+if __name__ == '__main__':
+    with get_channel() as channel:
+        print(' [*] Waiting for Messages. To exit press CTRL+C')
+
+        channel.basic_consume(message_handler, queue=DASHBOARD_QUEUE_NAME, no_ack=True)
```

Comment:
which part performs the consumer ack? Hidden in `start_consuming`?

Reply:
Hidden away, you can, however, pass `basic_consume` `no_ack=True` to disable the consumer ack if you wanted to. (its `False` by default)